src/armature/Node.ts

```diff
@@ -916,19 +914,19 @@ export class GeometryNode extends Node {
      * transformations multiplied by the `coordinateSpace` parameter.
      *
      * @param {mat4} coordinateSpace The coordinate space this node resides in.
-     * @param {boolean} makeBones Whether or not the armature heirarchy should be visualized.
+     * @param {boolean} _makeBones Whether or not the armature heirarchy should be visualized.
      * @returns {NodeRenderObject} The geometry for this armature subtree, and possibly geometry
      * representing the armature itself.
      */
     public computeRenderInfo(
         coordinateSpace: mat4,
         normalTransform: mat3,
-        makeBones: boolean
+        _makeBones: boolean
     ): { currentMatrix: mat4; currentNormalMatrix: mat3; objects: NodeRenderObject } {
         const { currentMatrix, currentNormalMatrix, objects } = super.computeRenderInfo(
             coordinateSpace,
             normalTransform,
-            makeBones
+            false
```

Comment:
We're not using `_makeBones` anymore. Do we want to change the function signature to no longer include it, or update this statement to default to `false` if `_makeBones` is `undefined`?

Reply:
We still use it on `Node`, this is just `GeometryNode`'s override of that method. Because it's an override, I opted to keep the parameter around so the signature is the same, but just prefixed with the underscore.

Reply:
Oh, my bad, forgot about the abstractions we had 🤦‍♂️.